app/src/main/java/com/main/mealplanner/UI/MainFragment.kt

```diff
@@ -11,8 +11,6 @@ import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import com.main.mealplanner.MainViewModel
 import androidx.lifecycle.ViewModelProviders
-import androidx.recyclerview.widget.DefaultItemAnimator
-import androidx.recyclerview.widget.LinearLayoutManager
```

Comment:
Removed the unused imports so the code looks cleaner.
Source:
https://kotlinlang.org/docs/coding-conventions.html#verify-that-your-code-follows-the-style-guide